tests/test_helpers.py

```diff
@@ -191,20 +212,23 @@ def deboolize(lib):
         lib[key] = deboolized(value)
 
 
-def normalize_ufo_lib(path):
+def normalize_ufo_lib(path, ufo_module):
     """Go through each `lib` element recursively and transform `bools` into
     `int` because that's what's going to happen on round-trip with Glyphs.
     """
-    font = defcon.Font(path)
+    try:
+        font = ufo_module.Font.open(path)
+    except AttributeError:
+        font = ufo_module.Font(path)
```

Comment:
maybe it's worth it to add `ufo_module_open` and `ufo_module_save` helper functions to encapsulate the API diffs. Also makes it easier to search and replace when defcon support is dropped.